docs/notebook_converter.sh

```diff
@@ -1,9 +0,0 @@
-
```

Comment:
Why is this file removed? Was this file generating the notebooks for `readthedocs`?

Reply:
It wasn't actually used. The list of commands executed by readthedocs (see https://github.com/entropicalabs/openqaoa/blob/feat_fix_docs/.readthedocs.yml#L17) and the action that tests the docs is already doing it anyway (see https://github.com/entropicalabs/openqaoa/blob/feat_fix_docs/.github/workflows/test_dev.yml#L108)